Zotero/Scenes/General/Views/SyncToolbarController.swift

```diff
@@ -194,6 +194,15 @@ final class SyncToolbarController {
                 case .notChanged: break // Should not happen
                 }
 
+            case .webDavUpload(let error):
+                switch error {
+                case .cantCreatePropData: break // Should not happen
+
+                case .apiError(let error):
+                    guard let statusCode = error.unacceptableStatusCode else { break }
+                    return (L10n.Errors.SyncToolbar.webdavRequestFailed(statusCode, error.url?.absoluteString ?? ""), nil)
```

Comment:
I thought that the second argument is the HTTP method, not the URL 🤔

Reply:
My bad, I forgot to change it after Dan replied